test/lender/Lender.sol

```diff
@@ -256,13 +253,15 @@ contract Lender {
       require(ITempus(principal).yieldBearingToken() == IErc20Metadata(u), 'tempus underlying != underlying');
       require(ITempus(principal).maturityTime() == m, 'tempus maturity != maturity');
 
-      // Transfer funds from user to Illuminate, Scope to avoid stack limit
+      // Get the underlying token
       IErc20 underlyingToken = IErc20(u);
+
+      // Transfer funds from user to Illuminate, Scope to avoid stack limit
       Safe.transferFrom(underlyingToken, msg.sender, address(this), a);
 
       // Swap on the Tempus Router using the provided market and params
       IZcToken illuminateToken = IZcToken(IIlluminate(illuminate).markets(u, m)[uint256(Illuminate.Principals.Illuminate)]);
-      uint256 returned = ITempus(tempusAddr).depositAndFix(Any(x), Any(t), a, true, r, d) - illuminateToken.balanceOf(address(this));
+      uint256 returned = ITempus(tempusAddr).depositAndFix(Any(x), Any(t), a - a / feenominator, true, r, d) - illuminateToken.balanceOf(address(this));
```

Comment:
Let's scope these calls so that we can maintain the same style throughout. This will likely get optimized out later on, but it's going to be good to have while we test out the contracts.